assets/scss/templates/_integrations.scss

```diff
@@ -0,0 +1,83 @@
+body {
+  &[data-active-menu="13"] {
+    background-color: #f6f6f6
```

Comment:
dont have variable for this color?

Reply:
We have a really close one, but not exactly the same.